cli_tools_e2e_test/gce_image_import_export/test_suites/import/post_translate_test.sh

```diff
@@ -0,0 +1 @@
+../../../../daisy_integration_tests/scripts/post_translate_test.sh
```

Comment:
This is strange, you still need to copy both "cli_tools_e2e_test/.../post_translate_test.sh" and "daisy_integration_tests/.../post_translate_test.sh" to the docker, and need to maintain the relative path in the docker. Why not just copy ""daisy_integration_tests/.../post_translate_test.sh" in the docker build file?

Reply:
Sounds good to me; removed the symlink in the latest commit.

The original purpose was for running the tests locally, but I've found it much easier (as you've suggested) to just use the docker container for running the tests.